content/applications/finance/fiscal_localizations/kenya.rst

```diff
@@ -7,114 +7,320 @@ Kenya
+company: :guilabel:`Insurance Code`, :guilabel:`Insurance Name`, and :guilabel:`Insurance Rate`.
+
+.. _kenya/product-registration:
+
+Items registration
+==================
+
+The KRA requires **products to be registered** first before business operations (such as stock
+movements, :abbr:`BOM (Bill of Materials)`, customer invoices, etc.). To be registered, each product
+must have the following elements defined. These can be found in the **product form view**
+(:menuselection:`Accounting --> Customers --> Products` or :menuselection:`Accounting --> Vendors
+--> Products`) by clicking a **product** under the :guilabel:`General Information` tab and the
+:guilabel:`Accounting` tab (in the **KRA eTIMS details** section).
+
+Under the :guilabel:`General information` tab:
+
+- :guilabel:`Cost`.
+
+Under the :guilabel:`Accounting` tab:
+
+- :guilabel:`Packaging Unit`;
+- :guilabel:`Packaging Quantity`;
+- :guilabel:`Origin Country`;
+- :guilabel:`eTIMS Product Type`;
+- :guilabel:`Insurance Applicable`;
+- :ref:`UNSPSC Category <etims/unspsc>`.
+
+Odoo checks to see if the elements above are defined when conducting business operations. If yes,
+the product is automatically registered while sending the operation to the KRA. If not, the user is
+alerted by a yellow banner at the top of the screen they are currently viewing, inviting them to
+check the missing elements by clicking :guilabel:`View Product(s)`.
+
+.. image:: kenya/product-registration.png
+   :alt: Product registration template.
+
+Stock movements
+===============
+
+All **stock movements** must be sent to the KRA. They do not require an invoice if they are
+**internal operations** or **stock adjustments**. Therefore, Odoo automatically sends them if the
+following conditions are met:
+
+#. No partners are set for the move; or
+#. The partner is your main company or a branch of the main company.
+
+If the stock moves are **external** (e.g., to someone external the main company or its branches),
+the stock moves are automatically sent *after* the invoice is sent to eTIMS.
+
+.. note::
+   - The stock move must be confirmed before sending the invoice to eTIMS;
+   - The product(s) must be registered for the stock move to be sent to eTIMS. If not, a yellow
+     banner appears to prompt registration.
+
+Purchases
+=========
+
+Odoo automatically fetches new vendor bills from eTIMS every day. Users need to confirm the fetched
+vendor bills and the confirmation to the KRA. Every bill confirmation must be associated with a
+validated receipt. To do so, each bill must be linked to purchase order lines.
+
+.. _kenya/purchases:
+
+In the case of purchases **(not customs imports)**, the steps to link purchase order lines with
+bills are the following:
+
+#. The vendor bill is fetched from the KRA servers. The JSON file is available in the chatter of the
+   vendor bill if needed;
+#. Odoo looks at the **PIN** (tax ID) of the partner;
```

Comment:
I'm guessing they use PIN instead of Tax ID because of local fiscal reasons which is not a tax ID but rather a PIN. In any case, i changed it but left pin between brackets